docs/usage/faq.md

```diff
@@ -36,6 +36,26 @@ The maintainers don't follow any release schedule or release cadence.
 This means the Mend Renovate App can lag a few hours to a week behind the open source version.
 Major releases of Renovate are held back until the maintainers are reasonably certain it works for most users.
 
+## How can I see which version the Mend Renovate app is using?
+
+Follow these steps to see which version the Mend Renovate app is on:
+
+1. Go to the [Mend Developer Portal](https://developer.mend.io/)
+1. Sign in to the Renovate app with your GitHub or Bitbucket account
+1. Select the app you would like to log in with: Forking Renovate or Renovate
```

Comment:
```suggestion
1. Select the app you would like to see .....: Forking Renovate or Renovate
```

The sign in step happens _before_ the "Select the app" screen, so I'm dropping the `would like to log in with:` part.

I need help with the wording of the improved instruction:

- Select the app you would like to _see the management screen_ for: Forking Renovate or Renovate
- Select the app you would like to _see the logs_ for: Forking Renovate or Renovate
- Something else

**Maintainers**: tell me what you like best. :wink:

Reply:
Signin happens after the selection actually

Reply:
Oh okay. I'll keep what we have now. :smile: